src/connector/src/sink/coordinate.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::{Instant, SystemTime, UNIX_EPOCH};
+
+use anyhow::anyhow;
+use chrono::Utc;
+use risingwave_common::array::StreamChunk;
+use risingwave_common::buffer::Bitmap;
+use risingwave_pb::connector_service::sink_writer_to_coordinator_msg::{
+    CommitRequest, StartCoordinationRequest,
+};
+use risingwave_pb::connector_service::{
+    sink_coordinator_to_writer_msg, sink_writer_to_coordinator_msg, SinkCoordinatorToWriterMsg,
+    SinkMetadata, SinkWriterToCoordinatorMsg,
```

Comment:
Where are the definition of `SinkWriterToCoordinatorMsg` and `SinkCoordinatorToWriterMsg`?

Reply:
Oops. This file is included by mistake. It does not belong to this PR. 

The definition of them are in PR https://github.com/risingwavelabs/risingwave/pull/10757